ui/app/templates/components/auth-info.hbs

```diff
@@ -68,6 +63,18 @@
               </li>
             {{/if}}
           {{/if}}
+
+          <li class="action">
+            <button
+              type="button"
+              class="link button"
+              disabled={{this.registerWebauthn.isRunning}}
+              onclick={{perform this.registerWebauthn}}
+            >
+              Register WebAuthn
+            </button>
+          </li>
```

Comment:
The button will show up in the account dropdown on the first login page. Seemed as good a place as any for now 🤷 

<img width="1219" alt="Screen Shot 2022-10-19 at 2 16 01 PM" src="https://user-images.githubusercontent.com/46458931/196772176-01a9a356-f260-45b1-8456-f7eb513b712e.png">